src/components/PyatchSpriteName.jsx

```diff
@@ -34,7 +34,8 @@ export function PyatchSpriteName(props) {
             fullWidth
             sx={{my: "1vh", input: { color: 'white'}, fieldset: { borderColor: "white" }}}
             size="small"
+            disabled={targetIds[0] == editingTargetId}
         />
-        <Button variant="contained" color="success" onClick={handleSave} disabled={nameSaved} sx={{m:"1vh"}}><SaveIcon/></Button>
+        <Button variant="contained" color="success" onClick={handleSave} disabled={nameSaved || (targetIds[0] == editingTargetId)} sx={{m:"1vh"}}><SaveIcon/></Button>
```

Comment:
```suggestion
        <Button variant="contained" color="success" onClick={handleSave} disabled={nameSaved || editingTarget.isStage} sx={{m:"1vh"}}><SaveIcon/></Button>
```
Using this prop of the editing target makes it a little bit more explicit what you're checking